indico/MaKaC/review.py

```diff
@@ -2095,9 +2098,13 @@ def mergeInto(self,responsible,targetAbs,mergeAuthors=False,comments=""):
             #    if self.isPrimaryAuthor(auth):
             #        targetAbs.addPrimaryAuthor(newAuth)
             for auth in self.getPrimaryAuthorList():
+                if auth in targetAbs.getPrimaryAuthorList():
```

Comment:
`O(n²)`? ;) Why not create a set from that list outside the loop and check against that instead?